plugins/mcreator-core/procedures/is_mod_loaded.json

```diff
@@ -0,0 +1,21 @@
+{
+ "args0": [
+     {
+       "type": "input_value",
+       "name": "modid",
+       "check": "String"
+     }
+   ],
+   "inputsInline": true,
+   "output": "Boolean",
+   "colour": 250,

```

Comment:
This procedure returns a logic value, so use `"%{BKY_LOGIC_HUE}"` for the color here